modules/date.js

```diff
@@ -0,0 +1,6 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayDate() {
+  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  document.getElementById('date-time').textContent = now;
+}
```

Comment:
I can see that you made a good effort to get the date to show 💪 Unfortunately however, It doesn't show on the page as expected. I suggest you take a closer look at this bit of code and make sure that you obtained the luxon library properly 😉 
![image](https://user-images.githubusercontent.com/107795045/229944551-e1927ced-ddb3-48c1-96b2-d3d11b1b5644.png)